plugins/org.jkiss.dbeaver.core/OSGI-INF/l10n/bundle_fr.properties

```diff
@@ -137,4 +137,33 @@ confirm.driver_download.title = T\u00E9l\u00E9charger le pilote de base de donn\
 
 confirm.exit.message = Etes-vous s\u00FBr(e) ?
 
-confirm.exit.title = Quitter DBeaver
\ No newline at end of file
+confirm.exit.title = Quitter DBeaver
+content-type.org.jkiss.dbeaver.shortcut.name=Raccourci Windows
+view.shell.process.description=Suivre et contr\u00f4ler les processus shell en cours d'\u00E9x\u00E9cution
+command.org.jkiss.dbeaver.core.new.connection.from.url.name=Nouvelle connexion \u00e0 partir d'une URL JDBC
+command.org.jkiss.dbeaver.core.new.connection.from.url.description=Cr\u00E9er une nouvelle connexion \u00e0 partir d'une URL JDBC
+command.org.jkiss.dbeaver.core.new.connection.from.url.name_short=A partir d'une URL JDBC
+command.org.jkiss.dbeaver.core.navigator.bookmark.navigate.name=Naviguer Vers
+command.org.jkiss.dbeaver.core.navigator.bookmark.navigate.description=Acc\u00E9der \u00e0 un favori
+command.org.jkiss.dbeaver.core.disconnectOther.name=D\u00E9connecter les autres
+command.org.jkiss.dbeaver.core.disconnectOther.description=Fermer les connexions s\u00E9lectionn\u00E9es
+command.org.jkiss.dbeaver.core.disconnectProject.name=Se d\u00E9connecter du projet
+command.org.jkiss.dbeaver.core.disconnectProject.description=Fermer toutes les connexions du projet
+command.org.jkiss.dbeaver.core.connection.readonly.name=Lecture seule
+command.org.jkiss.dbeaver.core.connection.synchronize.name=Synchroniser
+command.org.jkiss.dbeaver.core.connection.synchronize.description=Synchronisez les sources de donn\u00E9es locales et distantes
+command.org.jkiss.dbeaver.core.qm.delete.name=Supprimer les requ\u00EAtes du journal
+command.org.jkiss.dbeaver.core.qm.delete.description=Supprimer les requ\u00EAtes s\u00E9lectionn\u00E9s du journal
+page.org.jkiss.dbeaver.preferences.editors.name=Editeurs
+page.org.jkiss.dbeaver.preferences.accessibility.name=Accessibilit\u00E9
+wizard.org.jkiss.dbeaver.core.wizard.export.workspace.name=Pr\u00E9f\u00E9rences
+wizard.org.jkiss.dbeaver.core.wizard.import.workspace.name=Pr\u00E9f\u00E9rences
+confirm.disable_network_handler.message=Etes vous sur de d\u00E9sactiver la configuration {0} ?
```

Comment:
```suggestion
confirm.disable_network_handler.message=Etes vous sur de d\u00E9sactiver la configuration {0}?
```

Reply:
In french, we have space between text and punctuation :)